src/idom/utils.py

```diff
@@ -309,6 +308,9 @@ def _vdom_attr_to_html_str(key: str, value: Any) -> tuple[str, str]:
     return key.lower(), str(value)
 
 
+# Pattern for delimitting camelCase names (e.g. camelCase to camel-case)
+_CAMEL_CASE_SUB_PATTERN = re.compile(r"(?<!^)(?=[A-Z])")
+
 # see list of HTML attributes with dashes in them:
 # https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes#attribute_list
-_DASHED_HTML_ATTRS = {"accept_charset", "http_equiv"}
+_DASHED_HTML_ATTRS = {"acceptCharset", "httpEquiv"}
```

Comment:
tbh these globals are innocent enough to be public instead of private.

It's possible some external libraries might want to rely on it some day, despite it being undocumented.